angularjs-google-maps/ng-map.d.ts

```diff
@@ -0,0 +1,148 @@
+﻿// Type definitions for angularjs-google-maps v1.17.3
+// Project: https://github.com/allenhwkim/angularjs-google-maps
+// Definitions by: Niko Kovačič <https://github.com/nkovacic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../angularjs/angular.d.ts" />
+/// <reference path="../googlemaps/google.maps.d.ts" />
+
+declare module "ngMap" {
+    let _: string;
+    export = _;
+}
+
+declare module ngMap {
```

Comment:
Can you change this to a `namespace` and rename the it so it matches the other angular definition files, like [angular-cookie](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/angular-cookie/angular-cookie.d.ts)